JuliaGPU/session1/Gauri/q3.jl

```diff
@@ -0,0 +1,11 @@
+#3. Printing ASCII values of each character in given string
+function printasc(str)
+        newstr = ""
+#    s = firstindex(str)
+    l = lastindex(str)
+    for i in 1:l
+        newstr*=string(Int(str[i]))
```

Comment:
This gives us the decimal representation. The questions ask for the hexadecimal representation.